src/utils/IConfig.ts

```diff
@@ -0,0 +1,10 @@
+export default interface IConfig {
+    readonly ERIC_PORT: number;
+    readonly CACHE_SERVER: string;
+    readonly CACHE_DB: number;
+    readonly CACHE_PASSWORD: string;
+    readonly COOKIE_NAME: string;
+    readonly COOKIE_SECRET: string;
+    readonly DEFAULT_SESSION_EXPIRATION: number;
```

Comment:
How is that used or intended to be used?

Reply:
the process.env give out variables of type `T  | undefined`. The point was centralise the checking of ENV variables so that we don't have to check every time if process.env variable is undefined